### PR TITLE
Feature/remove custom log

### PR DESCRIPTION
**Pull request by italolelis**
## What does this PR do?
- Removes the custom log file
- Adds init functions, this way it's cleaner when initializing something on the main package

### CODE REVIEW

File: cmd/janus/init.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+func init() {
+	dsn := globalConfig.StorageDSN
+	log.Debugf("Trying to connect to redis instance: %s", dsn)
+	redisStorage = redis.NewClient(&redis.Options{
+		Addr: dsn,
+	})
+}
+
+// initStatsd initializes new StatsD client if it enabled
+func init() {
```
Comment (kieranajp) [inline cmd/janus/init.go:62]: All the functions called `init` in the same file?

Comment (italolelis) [reply cmd/janus/init.go:62]: Yes, that's how go works with init functions. You can have multiple init function in the same package, they will be executed one after the other in the order that they show up.

**Comment by kieranajp** [comment]
:shipit:

[Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/35/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway)

**Comment by kieranajp** [comment]
:shipit:

[Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/35/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway)

**Pull request merged**